src/app/shared/components/image-cropper-modal/image-cropper-modal.component.html

```diff
@@ -13,6 +13,12 @@
       (imageCropped)="imageCropped($event)"
       (cropperReady)="cropperReady()"
       (loadImageFailed)="loadImageFailed()"></image-cropper>
+    <div *ngIf="invalidMinRequirements">
+      <small class="warning"
+        >Зображення недостатнього розміру.<br />Мінімальні розміри: ширина {{ data.cropperConfig.cropperMinWidth }}px, висота
```

Comment:
add locale, keep `>` at the end of the last line

Reply:
Merged by mistake